runtime/src/main/java/org/corfudb/runtime/view/replication/ChainReplicationProtocol.java

```diff
@@ -31,9 +32,9 @@ public ChainReplicationProtocol(IHoleFillPolicy holeFillPolicy) {
      * {@inheritDoc}
      */
     @Override
-    public void write(Layout layout, ILogData data) throws OverwriteException {
+    public void write(EpochedClient epochedClient, ILogData data) throws OverwriteException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.corfudb.runtime.exceptions.OverwriteException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)